strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml

```diff
@@ -77,6 +77,8 @@
     <logger name="org.carlspring.strongbox.data.server" level="DEBUG"/>
 
     <logger name="net.sf.ehcache" level="ERROR"/>
+
+    <logger name="org.apache.commons.beanutils" level="ERROR"/>
```

Comment:
Lots of debug noise since some time, unintentional I believe...

Reply:
+1

Reply:
+1